slack_cli_hooks/hooks/get_hooks.py

```diff
@@ -8,7 +9,7 @@
 )
 
 PROTOCOL: Protocol
-EXEC = "python3"
+EXEC = sys.executable or "python3"
```

Comment:
If the absolute path to python command has white spaces, the execution can fail:

```
$ mkdir "foo bar"
$ cd ./"foo bar"/

$ pwd
/Users/kazuhiro.sera/foo bar
$ cp -p /Users/kazuhiro.sera/.pyenv/versions/3.11.2/bin/python .

$ ./python
Python 3.11.2 (main, Mar  9 2023, 14:03:12) [Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.executable
'/Users/kazuhiro.sera/foo bar/python'
>>>
```

To prevent the error, having single quotes (or something similar) would be safer:
```suggestion
EXEC = f"'{sys.executable}'" or "python3"
```

Reply:
This is an amazing catch 🙇 for an edge case, I've tested this solution in `zsh` and `PowerShell` and everything works as expected 💯 Ill add a comment explaining why we need to stringy `sys.executable`